ui/app/templates/vault/cluster/requests.hbs

```diff
@@ -6,8 +6,4 @@
   </p.levelLeft>
 </PageHeader>
 
-{{#if (gt model.counters.length 1) }}
-  <HttpRequestsBarChart @counters={{model.counters}}/>
-{{/if}}
-<AlertInline @type="info" @message="Metrics are recorded only for interactions that produce or use a Vault token." />
-<HttpRequestsTable @counters={{model.counters}} />
```

Comment:
Was this removed to make development easier?

Reply:
agh, yes. i keep forgetting about this -- my bad. thank you for noticing!

Reply:
Oops, sorry I missed that!